packages/core/admin/admin/src/pages/Auth/components/Login.tsx

```diff
@@ -103,90 +111,43 @@ const Login = ({ children }: LoginProps) => {
-                          })
-                        : ''
-                    }
-                    onChange={handleChange}
-                    value={values.password}
-                    label={formatMessage({
-                      id: 'global.password',
-                      defaultMessage: 'Password',
-                    })}
-                    name="password"
-                    type={passwordShown ? 'text' : 'password'}
-                    endAction={
-                      <FieldActionWrapper
-                        onClick={(e) => {
-                          e.stopPropagation();
-                          setPasswordShown((prev) => !prev);
-                        }}
-                        label={formatMessage(
-                          passwordShown
-                            ? {
-                                id: 'Auth.form.password.show-password',
-                                defaultMessage: 'Show password',
-                              }
-                            : {
-                                id: 'Auth.form.password.hide-password',
-                                defaultMessage: 'Hide password',
-                              }
-                        )}
-                      >
-                        {passwordShown ? <Eye /> : <EyeStriked />}
-                      </FieldActionWrapper>
-                    }
-                    required
-                  />
-                  <Checkbox
-                    onValueChange={(checked) => {
-                      handleChange({ target: { value: checked, name: 'rememberMe' } });
-                    }}
-                    value={values.rememberMe}
-                    aria-label="rememberMe"
-                    name="rememberMe"
-                  >
-                    {formatMessage({
-                      id: 'Auth.form.rememberMe.label',
-                      defaultMessage: 'Remember me',
-                    })}
-                  </Checkbox>
-                  <Button fullWidth type="submit">
-                    {formatMessage({ id: 'Auth.form.button.login', defaultMessage: 'Login' })}
-                  </Button>
-                </Flex>
-              </Form>
-            )}
-          </Formik>
+            <Flex direction="column" alignItems="stretch" gap={6}>
+              {[
+                {
+                  label: formatMessage({ id: 'Auth.form.email.label', defaultMessage: 'Email' }),
+                  name: 'email',
+                  placeholder: formatMessage({
+                    id: 'Auth.form.email.placeholder',
+                    defaultMessage: 'kai@doe.com',
+                  }),
+                  required: true,
+                  type: 'string' as const,
+                },
+                {
+                  label: formatMessage({
+                    id: 'global.password',
+                    defaultMessage: 'Password',
+                  }),
+                  name: 'password',
```

Comment:
I think we need to mark this as required? That is one of the failing FE tests packages/core/admin/admin/src/pages/Auth/components/tests/Login.test.tsx

Reply:
⭐